eng/scripts/Create-Apiview-Token-Files.ps1

```diff
@@ -0,0 +1,40 @@
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $true)]
+  [string]$ReviewDetailsJson,
+  [Parameter(Mandatory = $true)]
+  [string]$StagingPath,
+  [Parameter(Mandatory = $true)]
+  [string]$WorkingDir,
+  [Parameter(Mandatory = $true)]
+  [string]$StorageBaseUrl,
+  [Parameter(Mandatory = $true)]
+  [string]$ContainerSas,
+  [Parameter(Mandatory = $true)]
+  [string]$Language
+)
+
+
+Write-Host "Review Details Json: $($ReviewDetailsJson)"
+$reviews = ConvertFrom-Json $ReviewDetailsJson
+if ($reviews -ne $null)
+{
+    foreach($r in $reviews)
+    {
+        $codeDir = New-Item -Path $WorkingDir/$($r.ReviewID)/$($r.RevisionID) -ItemType Directory
+        $sourcePath = $StorageBaseUrl + "/" + $r.FileID + $ContainerSas
+        azcopy cp "$sourcePath" $codeDir/$($r.FileName) --recursive=true
+
+        #Create staging path for review and revision ID 
+        New-Item -Path $StagingPath/$($r.ReviewID)/$($r.RevisionID) -ItemType Directory
+        if( $Language -eq 'Python')
+        {
+            python -m apistub --pkg-path $codeDir/$($r.FileName) --out-path $StagingPath/$($r.ReviewID)/$($r.RevisionID)
```

Comment:
As we chatted we don't want to embed all the language specific steps in one script we should refactor to have some common logic that language specific pipelines can use instead.